static/app/views/settings/projectPerformance/projectPerformance.tsx

```diff
@@ -235,6 +237,14 @@ class ProjectPerformance extends AsyncView<Props, State> {
     return fields;
   }
 
+  get areAllConfigurationsDisabled(): boolean {
+    let result = true;
+    Object.values(DetectorConfigAdmin).forEach(threshold => {
+      result = result && !this.state.performance_issue_settings[threshold];
+    });
+    return result;
```

Comment:
Nit: we can do this more semantically with:
```suggestion
    return Object.values(DetectorConfigAdmin).every(threshold => !this.state.performance_issue_settings[threshold]);
```

Reply:
Yup, I like this.

Reply:
I'll add it in a following PR.